### PR TITLE
Add -verrors=spec switch

### PR DESCRIPTION
**Pull request by WalterBright**
I have needed this occasionally, I expect others do too. Prints messages out in lovely magenta.

### CODE REVIEW

File: changelog.dd [hunk 1]
```diff
@@ -25,6 +25,7 @@ $(BUGSTITLE Compiler Changes,
     $(LI $(RELATIVE_LINK2 native_tls_osx, Native TLS on OS X 64 bit.))
     $(LI $(RELATIVE_LINK2 __FILE_FULL_PATH__, Special keyword replaced by the source file's absolute file name.)
     $(LI $(RELATIVE_LINK2 dash_safe, Add -transition=safe switch.))
+    $(LI $(RELATIVE_LINK2 dash_verrors_spec, Add -verrors=spec switch.))
```
Comment (andralex) [inline changelog.dd]: surround -verrors=spec with backticks

File: changelog.dd [hunk 2]
```diff
@@ -183,6 +184,22 @@ $(BUGSTITLE Compiler Changes,
 	$(LI Prevents dereferencing `array.ptr` because that bypasses array bounds check.)
 	)
     )
+
+    $(LI $(LNAME2 dash_verrors_spec, Add -verrors=spec switch.)
```
Comment (andralex) [inline changelog.dd]: surround -verrors=spec with backticks

File: changelog.dd [hunk 3]
```diff
@@ -183,6 +184,22 @@ $(BUGSTITLE Compiler Changes,
...
+        {
+            int p;
+            bool b = __traits(compiles, { p = &i; });
+        }
+        ---
+        $(P that are normally not shown:)
+        ---
+        (spec:1) test.d(13): Error: cannot implicitly convert expression (& i) of type int* to int
+        ---
+        $(P The number after `spec:` is the nesting of the speculative compiles.)
```
Comment (andralex) [inline changelog.dd]: like here!

**Review by andralex — COMMENTED** [review]
Rule of thumb: if it's not English it should be formatted differently than plain text.

File: changelog.dd (hunk 1 above)
Comment (WalterBright) [inline changelog.dd]: done

File: changelog.dd (hunk 2 above)
Comment (WalterBright) [inline changelog.dd]: done

**Pull request merged**